src/vs/workbench/contrib/issue/electron-browser/issue.contribution.ts

```diff
@@ -19,8 +19,29 @@ const helpCategory = nls.localize('help', "Help");
 const workbenchActionsRegistry = Registry.as<IWorkbenchActionRegistry>(Extensions.WorkbenchActions);
 
 if (!!product.reportIssueUrl) {
-	workbenchActionsRegistry.registerWorkbenchAction(new SyncActionDescriptor(OpenIssueReporterAction, OpenIssueReporterAction.ID, OpenIssueReporterAction.LABEL), 'Help: Open Issue Reporter', helpCategory);
 	workbenchActionsRegistry.registerWorkbenchAction(new SyncActionDescriptor(ReportPerformanceIssueUsingReporterAction, ReportPerformanceIssueUsingReporterAction.ID, ReportPerformanceIssueUsingReporterAction.LABEL), 'Help: Report Performance Issue', helpCategory);
+
+	const OpenIssueReporterActionId = 'workbench.action.openIssueReporter';
+	const OpenIssueReporterActionLabel = nls.localize({ key: 'reportIssueInEnglish', comment: ['Translate this to "Report Issue in English" in all languages please!'] }, "Report Issue");
+
+	CommandsRegistry.registerCommand(OpenIssueReporterActionId, function (accessor, args?: [string]) {
+		let extensionId: string | undefined;
+		if (args) {
+			[extensionId] = args;
+		}
+
+		return accessor.get(IWorkbenchIssueService).openReporter({ extensionId });
+	});
+
+	const command: ICommandAction = {
+		id: OpenIssueReporterActionId,
+		title: { value: OpenIssueReporterActionLabel, original: 'Help: Open Issue Reporter' },
+		category: helpCategory
+	};
+
+	MenuRegistry.addCommand(command);
+
+	MenuRegistry.appendMenuItem(MenuId.CommandPalette, { command });
```

Comment:
Pretty sure that this is the same as calling `MenuRegistry.addCommand`

Reply:
I didn't see the command in the palette when this was removed, so I think this is necessary. However, the menubar already calls `MenuRegistery.addCommand` for this command, so I removed that